### PR TITLE
* Model Injection

### PR DESCRIPTION
**Pull request by biom4st3r**
directing added and replacing models at runtime

### CODE REVIEW

File: src/main/java/io/github/fabriccommunity/everything/annotations/Nullable.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Target maybe or return null
+ */
+@Target({FIELD,METHOD,PARAMETER})
+@Retention(RetentionPolicy.CLASS)
+public @interface Nullable {
```
Comment (i509VCB) [inline src/main/java/io/github/fabriccommunity/everything/annotations/Nullable.java:14]: Ah yes so we have 2 null annos now

Comment (biom4st3r) [reply src/main/java/io/github/fabriccommunity/everything/annotations/Nullable.java:14]: it's good to give developers options to pick from

Comment (vini2003) [reply src/main/java/io/github/fabriccommunity/everything/annotations/Nullable.java:14]: entirely agreed, will merge in the next 30 minutes if nobody complains
the race is on!

Comment (valoeghese) [reply src/main/java/io/github/fabriccommunity/everything/annotations/Nullable.java:14]: nice

File: src/main/resources/everything.mixins.json [hunk 2]
```diff
@@ -6,7 +6,9 @@
   "mixins": [
   ],
   "client": [
-    "ExampleMixin"
+    "ExampleMixin",
+    "modeinjection/BlockModelsMixin",
+    "modeinjection/ItemModelsMixin"
```
Comment (valoeghese) [inline src/main/resources/everything.mixins.json:11]: seems fine yes merge

**Pull request merged**